react/src/Containers/App.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Route,
+  Link
+} from 'react-router-dom'
+
+import Homepage from './Homepage'
+
+import RestaurantsIndexContainer from './RestaurantsIndexContainer'
+import RestaurantShowContainer from './RestaurantShowContainer'
+import RestaurantsNewContainer from './RestaurantsNewContainer'
+import RestaurantEditContainer from './RestaurantEditContainer'
```

Comment:
Careful about adding containers before you actually are using them. It's best to only import things once you're using them so you don't have unnecessary code!